src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -36,7 +36,10 @@ public class DocumentationBrowserViewExtension : IViewExtension, ILogSource
         /// </summary>
         public bool AllowRemoteResources
         {
-            get => this.allowRemoteResources;
+            get
+            {
+                return this.allowRemoteResources;
+            }
             set
```

Comment:
hmm, what about making this internal?

Reply:
It is used in the tests. Are you suggesting to use [InternalsVisibleTo](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=netframework-4.8)?

Maybe I'm missing something, but I don't see a way to enable this in normal usage. Won't this be always disabled?

Reply:
yes, I don't think we should encourage any extensions authors to enable this, potentially removing it completely.

Reply:
Ok. I'll remove the feature then.

Reply:
well, open to your thoughts on it, want to proceed carefully with potential redirection to external links.

Reply:
I have no problem with it :) I actually agree it's the safest approach. There is the security aspect of it but also with embedded HTML you can always get help, even if offline. We could say we are enforcing a good practice and people can always add links to additional online resources, if needed.